File: DiscoverHSCountry.Services/Database/PublicCityService.cs
using System;
using System.Collections.Generic;

namespace DiscoverHSCountry.Services.Database;

public partial class PublicCityService
{
    public int PublicCityServiceId { get; set; }

    public string Name { get; set; } = null!;

    public string Description { get; set; } = null!;

    public string? CoverImage { get; set; }

    public string Address { get; set; } = null!;

    public int? CityId { get; set; }

    public virtual City? City { get; set; }
}

File: UI/discoverhscountry_desktop/lib/models/public_city_service.dart
import 'package:json_annotation/json_annotation.dart';
part 'public_city_service.g.dart';


@JsonSerializable()
class PublicCityService {
  int? publicCityServiceId;
  String name;
  String description;
  String? address;
  int? cityId;
  String? coverImage;

  PublicCityService({
    this.publicCityServiceId,
    required this.name,
    required this.description,
    this.address,
    this.coverImage,
    this.cityId,
  });
   factory PublicCityService.fromJson(Map<String, dynamic> json) =>
      _$PublicCityServiceFromJson(json);

  Map<String, dynamic> toJson() => _$PublicCityServiceToJson(this);
  
  }